modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -105,6 +108,20 @@ export default class MultiIconLayer extends IconLayer {
       value[i++] = (rect.height / 2) * getAnchorY(object) || 0;
     }
   }
+
+  calculateInstancePickingColors(attribute) {
+    const {data, getPickingIndex} = this.props;
+    const {value} = attribute;
+    let i = 0;
+    for (const point of data) {
+      const index = getPickingIndex(point);
+      const pickingColor = this.encodePickingColor(index);
```

Comment:
This will break the current picking logic. You need to override the default `getPickingInfo` as well.